skimage/meson.build

```diff
@@ -29,37 +29,51 @@ if is_windows
   endif
 endif
 
-# NumPy include directory - needed in all submodules
-incdir_numpy = run_command(py3,
-  [
-    '-c',
-    'import os; os.chdir(".."); import numpy; print(numpy.get_include())'
-  ],
-  check: true
-).stdout().strip()
+# Both NumPy and Pythran require header files that may differ between the build
+# system and the host system in a cross-compilation environment. To accommodate
+# these cases, we can query user-defined properties that can be specified in
+# the 'properties' section of a Meson cross file:
+#
+#   [properties]
+#   numpy-include-dir = '/path/to/numpy/includes'
+#   pythran-include-dir = '/path/to/pythran/includes'
+#
+# In the absence of explicitly configured paths, just run the build Python and
```

Comment:
I don't understand this docstring comment; could we target this for a developer of the library who doesn't work on systems such as conda-forge? E.g., what is build vs host system, and how is it relevant to how we build the library?

E.g., not sure if this is correct, but you could say something like:

> When cross-compiling skimage, the headers for the target platform may differ from those on the build system. To ensure that the build happens against the correct headers, their location is given in the `properties` section of the Meson build file:
>
> ...
>
> Try the properties file first, and if it doesn't exist, ask NumPy/Pythran for their header paths:

Reply:
I revised the text to better describe the issue and included a link to the Meson documentation that defines the major concepts. Let me know if anything else needs clarification.